geraldo/generators/csvgen.py

```diff
@@ -90,7 +90,7 @@ def generate_csv(self):
         # First row with column names
         if self.first_row_with_column_names:
             cells = [(col.name or col.expression or col.attribute_name) for col in columns]
-            self.writer.writerow(cells)
+            self.writer.writerow([cell.encode("utf-8") for cell in cells])
```

Comment:
Does this issue only crop up with csv, or is pdf affected too?

Reply:
Yea, it seems to only happen with csv. Tested pdf locally and works.